packages/gatsby/cache-dir/find-path.js

```diff
@@ -17,6 +17,16 @@ const trimPathname = rawPathname => {
   return trimmedPathname
 }
 
+function absolutify(path) {
+  // If it's already absolute, return as-is
+  if (!/^\.{1,2}\/(?!\/)/.test(path)) {
+    return path
+  }
+  // Calculate path relative to current location, adding a trailing slash to
+  // match behavior of @reach/router
+  return new URL(path, window.location.href.replace(/([^/])$/, `$1/`)).pathname
```

Comment:
URL constructor is not supported in IE11. https://caniuse.com/#search=url, does polyfiling take care of that for us?

Reply:
Yeah, I wondered about that, but our docs says that we have babel auto-polyfills so I guessed it was ok